portal/models/fhir.py

```diff
@@ -25,6 +25,7 @@
 from .organization import OrgTree
 from .performer import Performer
 from .reference import Reference
+from past.builtins import basestring
```

Comment:
Can you move this import to the 1st block of imports, after the `flask` import line, to keep it alphabetical?


Imports are typically separated into 3 sorted blocks:
1. Built-in modules (eg `os`)
2. 3rd party modules (typically installed from PyPI)
3. Submodules in the same project

We're trying a tool out ([isort](https://github.com/timothycrosley/isort)) if you want to give it a try too